spec/factories/users.rb

```diff
@@ -1,6 +1,14 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    # don't freak out if the database isn't created yet
+    # http://stackoverflow.com/a/27745684/358804
```

Comment:
Startlingly similar to https://github.com/18F/tock/pull/161 . The inclusion of a file probably shouldn't force a DB call. Does this have the same semantics?

```patch
diff --git a/spec/factories/users.rb b/spec/factories/users.rb
index 7916d8a..5c4d0d5 100644
--- a/spec/factories/users.rb
+++ b/spec/factories/users.rb
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :user do
-    sequence(:email_address, User.count) {|n| "liono#{n}@some-cartoon-show.com" }
+    sequence(:email_address) {|n| "liono#{n}@some-cartoon-show.com" }
     first_name "Liono"
     last_name "Thunder"
```